assembly/onpremises-ide-packaging-war-platform-api/src/main/java/com/codenvy/api/deploy/OnPremisesIdeApiModule.java

```diff
@@ -469,5 +470,15 @@ protected void configure() {
         install(new SystemLicenseModule());
 
         bind(SystemLicenseWorkspaceFilter.class);
+
+        MapBinder<String, org.eclipse.che.plugin.docker.machine.ServerEvaluationStrategy> strategies =
+                MapBinder.newMapBinder(binder(),
+                                       String.class,
+                                       org.eclipse.che.plugin.docker.machine.ServerEvaluationStrategy.class);
+        strategies.addBinding("codenvy")
```

Comment:
Mb it would be better to move `"codenvy"` in constant of `CodenvyDockerServerEvaluationStrategy`?